app/components/logic/pretty-print-time.tsx

```diff
@@ -13,7 +13,13 @@ export const PrettyPrintTime: React.FunctionComponent<PrettyPrintTimeProps> = ({
     time ? new Date(time).toISOString() : "None"
   );
   useEffect(
-    () => setTimeString(time ? new Date(time).toLocaleString() : "None"),
+    () =>
+      setTimeString(
+        (time ? new Date(time).toLocaleString() : "None") +
```

Comment:
If `time` is undefined and the ternary resolves to false, then I think this resolves to "None (local time), None ago." I think instead we could put your additions just on the true case inside this ternary -- so that way we get just "None" when it's undefined